scripts/deploy.sh

```diff
@@ -30,6 +30,11 @@ function run_ansible {
 # begin the bootstrap process
 cd ${OA_DIR}
 
+# bootstrap ansible and install galaxy roles (needed whether AIO or multinode)
+which openstack-ansible || ./scripts/bootstrap-ansible.sh
+ansible-galaxy install --role-file=/opt/rpc-openstack/ansible-role-requirements.yml --force \
+                           --roles-path=/opt/rpc-openstack/rpcd/playbooks/roles
+
```

Comment:
I'm not aware of a reason to keep these out of `/etc/ansible/roles`. Were we using a tool like ansigenome to generate dependency graphs, all roles would need to be in the same directory since it only operates on a single one, but we're not, and that can be added as part of a separate docs build job if one is ever made.